sphinx/tutorial/rose/furthertopics/rose-stem-tutorial.rst

```diff
@@ -129,12 +129,12 @@ The ``$SOURCE_SPACESHIP`` environment variable will be set using the
 Jinja2 variable of the same name which is provided by Rose Stem.
 
 
```

Comment:
Is the rose-stem tutorial being deliberately kept?

Reply:
Yep, `rose stem` isn't dead, it's just awaiting its new life in cylc-rose - https://github.com/cylc/cylc-rose/pull/45

I guess makes sense if the docs remain here since it's a rose cmd.